lib/Doctrine/ORM/Tools/Console/Command/EnsureProductionSettingsCommand.php

```diff
@@ -70,7 +70,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             if ($input->getOption('complete') !== null) {
                 $em->getConnection()->connect();
             }
-        } catch (\Exception $e) {
+        } catch (\Throwable $e) {
+            $output->writeln('<error>' . $e->getMessage() . '</error>');
```

Comment:
this probably doesn't have to be cought, symfony console should render the error as it's not normal failure